modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectFolderExportImportTest.java

```diff
@@ -48,6 +48,7 @@ public class ObjectFolderExportImportTest extends BaseExportImportTestCase {
 			PermissionCheckerMethodTestRule.INSTANCE);
 
 	@Before
+	@Override
 	public void setUp() throws Exception {
 		user = TestPropsValues.getUser();
```

Comment:
@devpedrotavares reuse the existing `user`.. you don't need to do `user = TestPropsValues.getUser();`